packages/devtools_app/lib/src/common_widgets.dart

```diff
@@ -122,6 +122,7 @@ class IconLabelButton extends StatelessWidget {
     return SizedBox(
       height: defaultButtonHeight,
```

Comment:
add `width: !includeText(context, includeTextWidth) ? buttonMinWidth : null,` here, and with the other changes suggested in this round of review comments, this should solve the extra space around the button